api/tests/unit/domain/usecases/save-admin-member_test.js

```diff
@@ -0,0 +1,89 @@
+const { expect, sinon, catchErr, domainBuilder } = require('../../../test-helper');
+const saveAdminMember = require('../../../../lib/domain/usecases/save-admin-member');
+const { ROLES } = require('../../../../lib/domain/constants').PIX_ADMIN;
+const { AlreadyExistingAdminMemberError, UserNotFoundError, NotFoundError } = require('../../../../lib/domain/errors');
+const AdminMember = require('../../../../lib/domain/models/AdminMember');
+
+describe('Unit | UseCase | save-admin-member', function () {
+  context('when admin member email is not found', function () {
```

Comment:
Si ce test couvre le cas où l'email n'existe pas en BDD, je préfèrerai simplifier
```suggestion
  context('when email is not found', function () {
```